src/ResourceManager/DataLakeAnalytics/Commands.DataLakeAnalytics/Commands/NewAzureRmDataLakeAnalyticsCatalogSecret.cs

```diff
@@ -55,7 +55,7 @@ public class NewAzureDataLakeAnalyticsCatalogSecret : DataLakeAnalyticsCmdletBas
 
         [Parameter(ValueFromPipelineByPropertyName = true, ParameterSetName = BaseParameterSetName, Position = 3,
             Mandatory = true, HelpMessage = "The host of the database to connect to in the format 'myhost.dns.com'.")]
-        public string Host { get; set; }
+        public string DatabaseHost { get; set; }
```

Comment:
Please add an alias to the previous parameter name to avoid a breakign change.  Also, be sure to avoid any breaking changes in the properties of the output type for this cmdlet as well.